packages/contracts-bedrock/test/L1/OPContractsManager.t.sol

```diff
@@ -473,34 +481,48 @@ contract OPContractsManager_ChainIdToBatchInboxAddress_Test is Test, FeatureFlag
 
 /// @title OPContractsManager_AddGameType_Test
 /// @notice Tests the `addGameType` function of the `OPContractsManager` contract.
-contract OPContractsManager_AddGameType_Test is OPContractsManager_TestInit {
-    function setUp() public virtual override {
-        super.setUp();
-
-        // Skip AddGameType tests when V2 dispute games are enabled
-        // TODO(#17260): Remove skip when V2 dispute game support for addGameType implemented
-        skipIfDevFeatureEnabled(DevFeatures.DEPLOY_V2_DISPUTE_GAMES);
-    }
-
+contract OPContractsManager_AddGameType_Test is OPContractsManager_TestInit, DisputeGames {
```

Comment:
If you want to make this marginally simpler, can also just make `OPContractsManager_TestInit` inherit from `DisputeGames` so we don't need to inherit everywhere else

Reply:
Done.